### PR TITLE
[Documentation] Configure Test report even on successful tests

### PR DESCRIPTION
**Pull request by jr0cket**
A test report is never created when all tests run are successful.  Add information on how to configure CIDER to create a test report when tests are successful.

Small documentation only PR, no tests required?

### CODE REVIEW

**Comment by jr0cket** [comment]
The test-lint is failing on a line not changed by this pull request.
Line 30 (previously line 25) is longer than the 80 character limit that the test-lint checks for.

**Comment by bbatsov** [comment]
We already have this info here https://docs.cider.mx/cider/testing/running_tests.html#display-test-report-on-success, but perhaps it's OK to duplicate it.

**Comment by jr0cket** [comment]
Oops, I read the running tests page too, I just missed that section of the page.  Documentation is hard :)

**Comment by jr0cket** [comment]
I've updated the PR with a link to the specific configuration section from the Test Reports page, to avoid duplication.

I have left the change to the first paragraph as only if you have read all the way down to almost the bottom of running tests page do you discover that the test report buffer only displays upon failure of tests.

Alternatively, you could make the whole configuration section from the Running Tests page a separate page and make the specific configuration items discoverable in the left hand menu navigation.

File: doc/modules/ROOT/pages/testing/test_reports.adoc [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
-After running your tests, CIDER displays a test result report in the
+Shoud tests fail then CIDER displays a test result report in the
 `+*cider-test-report*+` buffer. This buffer uses `cider-test-report-mode`,
 which makes it easy to review any failures that might have occurred
 and jump directly to the definition of failing tests.
 
+Calling test commands again will update the test report.
+
+link:running_tests.html#display-test-report-on-success[Configure test reports on success]
```
Comment (bbatsov) [inline doc/modules/ROOT/pages/testing/test_reports.adoc:11]: `link` is for external references, internally you should be using `xref`. I'd probably add `TIP: See also xref`, so it's consistent with the general style of the docs.

**Pull request merged**